src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -1,6 +1,6 @@
 package core.mate.academy.model;
 
-public abstract class Machine implements Workable {
+public abstract class Machine<T> implements Workable {
```

Comment:
it is unnecessary to do this
```suggestion
public abstract class Machine implements Workable {
```